src/si/dynamic_viscosity.rs

```diff
@@ -0,0 +1,98 @@
+//! Dynamic viscosity (base unit pascal second, kg · m⁻¹ · s⁻¹).
+
+quantity! {
+    /// Dynamic viscosity (base unit pascal second, kg · m⁻¹ · s⁻¹).
+    quantity: DynamicViscosity; "dynamic viscosity";
+    /// Dimension of dynamic viscosity, ML⁻¹T⁻¹ (base unit pascal second, kg · m⁻¹ · s⁻¹).
```

Comment:
Dimensions in ISQ order.

```suggestion
    /// Dimension of dynamic viscosity, L⁻¹MT⁻¹ (base unit pascal second, kg · m⁻¹ · s⁻¹).
```